src/main/java/com/tabnine/binary/fetch/BinaryVersionFetcher.java

```diff
@@ -27,6 +28,12 @@ public BinaryVersionFetcher(LocalBinaryVersions localBinaryVersions, BinaryRemot
      * @throws SecurityException, NoExistingBinaryException if something went wrong
      */
     public String fetchBinary() throws NoValidBinaryToRunException {
+        Optional<BinaryVersion> bootstrappedVersion = BootstrapperSupport.bootstrap_version(localBinaryVersions, binaryRemoteSource, binaryDownloader);
+        if (bootstrappedVersion.isPresent()) {
+            Logger.getInstance(getClass()).info(format("found local bootstrapped version %s", bootstrappedVersion.get()));
```

Comment:
debug level instead of info level

Reply:
the other log in the class is info for the same purpose